crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -326,7 +326,7 @@ impl ViewportBlueprint<'_> {
                         );
                         if re_ui
                             .small_icon_button(ui, &re_ui::icons::REMOVE)
-                            .on_hover_text("Remove group and all its children from the space view.")
+                            .on_hover_text("Remove group and all its children from the space view")
```

Comment:
Title case `Group` and `Space View` as well

Reply:
🤦🏻‍♂️